packages/react-dom/src/events/ReactDOMEventListener.js

```diff
@@ -94,17 +92,17 @@ function schedulerPriorityToLanePriority(
 ): LanePriority {
   switch (schedulerPriorityLevel) {
     case ImmediateSchedulerPriority:
-      return SyncLanePriority;
+      return DiscreteEventPriority;
     case UserBlockingSchedulerPriority:
-      return InputContinuousLanePriority;
+      return ContinuousEventPriority;
     case NormalSchedulerPriority:
     case LowSchedulerPriority:
       // TODO: Handle LowSchedulerPriority, somehow. Maybe the same lane as hydration.
-      return DefaultLanePriority;
+      return DefaultEventPriority;
     case IdleSchedulerPriority:
-      return IdleLanePriority;
+      return IdleEventPriority;
     default:
-      return NoLanePriority;
+      return DefaultEventPriority;
```

Comment:
This used to be NoLane, now it's DefaultLane. Any differences?

Reply:
I think that was just a mistake before. NoLanePriority would get passed to `findUpdateLane`, which would throw:

https://github.com/facebook/react/blob/25bfa287f6c49950d8ea16f542716835683442d6/packages/react-reconciler/src/ReactFiberLane.new.js#L544-L571

I suppose we never hit this internally because we don't have any native event handlers that aren't part of the big switch.

I'll add a regression test as a follow up.

Reply:
Actually I'll address this in the next PR. We're about to remove the LanePriority type in favor of returning a lane directly.